### PR TITLE
Idea dropdown replacement

### PR DESCRIPTION
**Pull request by muhtasimhafiz**
#### Description

Please include a short description of the suggested change and the reasoning behind the approach you have chosen.

#### Link to ticket

Please add a link to the GitHub issue being addressed by this change.

### Type

[] Fix
[] Feature
[] Cleanup 

#### Screenshot of the result

If your change affects the user interface, you should include a screenshot of the result with the pull request.

### CODE REVIEW

File: app/Domain/Goalcanvas/Templates/showCanvas.blade.php [hunk 1]
```diff
@@ -145,32 +145,44 @@ class="delete">
...
+                                            '<i class="fas fa-fw ' .
+                                            __($statusLabels[$filter['status']]['icon']) .
+                                            '"></i> ' .
+                                            $statusLabels[$filter['status']]['title'] .
+                                            ' ' .
+                                            __('links.view');
+                                    }
+                                @endphp
+
+                                <x-global::actions.dropdown :labelText="html_entity_decode($labelText)" class="btn dropdown-toggle" align="start"
```
Comment (marcelfolaron) [inline app/Domain/Goalcanvas/Templates/showCanvas.blade.php:166]: Can you update this to use <x-slot:label-Text> and remove the html_entity_decode?

File: app/Domain/Ideas/Templates/advancedBoards.blade.php [hunk 2]
```diff
@@ -178,42 +187,48 @@ class=""
...
+                                                            $labelText = '<span class="text">
+                                                                            <span id="userImage' . $row->id . '">
+                                                                                <img src="' . $userImageUrl . '" width="25" style="vertical-align: middle;" />
+                                                                            </span>
+                                                                            <span id="user' . $row->id . '"></span>
+                                                                         </span>';
+                                                        @endphp
+                                                        
+                                                        <x-global::actions.dropdown 
+                                                            :labelText="html_entity_decode($labelText)"
```
Comment (marcelfolaron) [inline app/Domain/Ideas/Templates/advancedBoards.blade.php:206]: Move into slot for label-text and remove html_entity_decode

File: app/Domain/Ideas/Templates/advancedBoards.blade.php [hunk 3]
```diff
@@ -291,7 +306,7 @@ class="dropdown ticketDropdown userDropdown noBg show right lastDropdown dropRig
                         <form action="" method="post">
                             <div class="modal-header">
                                 <button type="button" class="close" data-dismiss="modal"
-                                    aria-hidden="true">&times;</button>
+                                    aripa-hidden="true">&times;</button>
```
Comment (marcelfolaron) [inline app/Domain/Ideas/Templates/advancedBoards.blade.php:309]: typo?

File: app/Views/Templates/components/actions/dropdown/item.blade.php [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @props([
-    'variant' => 'link', // Variant of the dropdown item: 'link' or 'border'
+    'variant' => 'link', // Variant of the dropdown item: 'link', 'border', 'nav-header', or 'nav-header-border'
```
Comment (marcelfolaron) [inline app/Views/Templates/components/actions/dropdown/item.blade.php:2]: can we just keep the name the variants "header" and "border". the nav-XXX seems rather long and the border doesn't always have to be near a header :D ?

Comment (marcelfolaron) [reply app/Views/Templates/components/actions/dropdown/item.blade.php:2]: PS: Should have looked at the file first. 

header
border
header-border


Should be fine

**Review by marcelfolaron — CHANGES_REQUESTED** [review]

File: app/Domain/Ideas/Templates/advancedBoards.blade.php (hunk 3 above)
Comment (muhtasimhafiz) [reply app/Domain/Ideas/Templates/advancedBoards.blade.php:309]: fixed

**Review by marcelfolaron — APPROVED** [review]

**Pull request merged**